### PR TITLE
Taking out all of Jeremy's information from our Finch Docs

### PR DESCRIPTION
**Pull request by cathy-chi**

### CODE REVIEW

File: docs/Management/2-Introspect.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ curl https://api.tryfinch.com/introspect \
     "company",
     "directory"
   ],
-  "username": "jeremy@tryfinch.com",
+  "username": "janedoe@tryfinch.com",
```
Comment (aren55555) [inline docs/Management/2-Introspect.md:31]: nit: janedoe@example.com

Comment (cathy-chi) [reply docs/Management/2-Introspect.md:31]: Ah good call :)

File: docs/Employer/4 - Individual.md [hunk 2]
```diff
@@ -42,29 +42,29 @@ Parameter | Type | Required | Description
...
           {
-            "data": "jeremyziyuzhang@gmail.com",
+            "data": "janedoe@gmail.com",
             "type": "personal"
           }
         ],
         "phone_numbers": [
           {
-            "data": "4077313626",
+            "data": "4151234567",
```
Comment (aren55555) [inline docs/Employer/4 - Individual.md:60]: `xxx55501xx`
see: https://softwareengineering.stackexchange.com/questions/376535/whats-the-phone-number-equivalent-of-example-org

File: docs/Employer/4 - Individual.md [hunk 3]
```diff
@@ -42,29 +42,29 @@ Parameter | Type | Required | Description
...
         "id": "5d0b10a1-a09a-430f-81f1-20be735dc5e9",
-        "first_name": "Jeremy",
+        "first_name": "Jane",
         "middle_name": null,
-        "last_name": "Zhang",
+        "last_name": "Doe",
         "emails": [
           {
-            "data": "jeremy@tryfinch.com",
+            "data": "janedoe@tryfinch.com",
```
Comment (aren55555) [inline docs/Employer/4 - Individual.md:50]: nit: janedoe@work.example.com

File: docs/Employer/4 - Individual.md [hunk 4]
```diff
@@ -42,29 +42,29 @@ Parameter | Type | Required | Description
...
             "type": "work"
           },
           {
-            "data": "jeremyziyuzhang@gmail.com",
+            "data": "janedoe@gmail.com",
```
Comment (aren55555) [inline docs/Employer/4 - Individual.md:54]: nit: janedoe@home.example.com

**Review by aren55555 — APPROVED** [review]

**Comment by skilbjo** [comment]
End of an era

**Pull request merged**